src/coreclr/pal/src/eventprovider/lttngprovider/CMakeLists.txt

```diff
@@ -62,9 +62,20 @@ add_dependencies(coreclrtraceptprovider generated_eventing_headers)
 
 find_library(LTTNG NAMES lttng-ust)
 
-target_link_libraries(coreclrtraceptprovider
-                      ${LTTNG}
-)
+if(NOT CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
+  target_link_libraries(coreclrtraceptprovider
+                        ${LTTNG}
+  )
+else()
+  target_link_libraries(coreclrtraceptprovider
+                        lttng-ust-tracepoint
+                        urcu
+                        urcu-bp
+                        urcu-cds
+                        urcu-common
+                        ${LTTNG}
+  )
+endif()
```

Comment:
Curious, what is the error if we don't make this change? lttng-ust-tracepoint is actually a soft dependency, i.e. this library gets loaded via `dlopen()` during the run time of application. Even the existing `target_link_libraries` is not needed, it is a leftover from past refactoring, we can just delete it altogether.

Reply:
> Curious, what is the error if we don't make this change? lttng-ust-tracepoint is actually a soft dependency, i.e. this library gets loaded via `dlopen()` during the run time of application. Even the existing `target_link_libraries` is not needed, it is a leftover from past refactoring, we can just delete it altogether.

Yes, I thinks so.
I check it within my machine now.

Reply:
Thanks for your review!
I had deleted the modification.  (I guess that it's old code from the mips64 when we porting it.)